app/Services/PgBackRestService.php

```
@@ -0,0 +1,474 @@
+    {
+        return 'coolify-'.Str::slug($this->database->name).'-'.$this->database->uuid;
+    }
+
+    /**
+     * Initialize pgBackRest for the database (one-time setup)
+     */
+    public function initialize(): void
+    {
+        // Check if already initialized
+        if ($this->backup->pgbackrest_stanza_created) {
+            Log::info('pgBackRest already initialized', ['database' => $this->database->uuid]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'initialize'. (undefined)

(StaticAccess)